### PR TITLE
Update global vars with env values.

### PR DESCRIPTION
**Pull request by andypham**
Update the global vars value with the environment variable values.

### CODE REVIEW

File: mac [hunk 1]
```diff
@@ -22,12 +22,12 @@ mysql_password="root"
 mysqldump="/Applications/MAMP/Library/bin/mysqldump"
 dropbox_folder="/Volumes/Cloud/Dropbox/"
 php="/Applications/MAMP/bin/php/php5.5.26/bin/php"
-php_cli="/usr/local/bin/php"
+php_cli=$(which php)
 phpmyadmin="http://localhost/phpMyAdmin/"
-github_email_account="guarinogabriel@gmail.com"
-github_username="guarinogabriel"
+github_email_account=$(git config -l | grep user.email=.* | cut -d "=" -f2)
```
Comment (guarinogabriel) [inline mac]: @andypham This command is not working:
`$(git config -l | grep user.email=.* | cut -d "=" -f2)`

It should be modified to:
`$(git config -l | grep user.email= | cut -d "=" -f2)`

File: mac [hunk 2]
```diff
@@ -22,12 +22,12 @@ mysql_password="root"
...
+github_username=$(git config -l | grep user.name=.* | cut -d "=" -f2)
```
Comment (guarinogabriel) [inline mac]: @andypham This command is not working:
`$(git config -l | grep user.name=.* | cut -d "=" -f2)`

It should be modified to:
`$(git config -l | grep user.name= | cut -d "=" -f2)`

File: mac [hunk 3]
```diff
@@ -22,12 +22,12 @@ mysql_password="root"
...
 echocommand="false"
-pref_editor="nano" # what's your preferred terminal editor?
+pref_editor=$EDITOR # what's your preferred terminal editor?
```
Comment (guarinogabriel) [inline mac]: @andypham This command is not returning anything. Could you please double check that?

File: mac [hunk 4]
```diff
@@ -22,12 +22,12 @@ mysql_password="root"
 mysqldump="/Applications/MAMP/Library/bin/mysqldump"
 dropbox_folder="/Volumes/Cloud/Dropbox/"
 php="/Applications/MAMP/bin/php/php5.5.26/bin/php"
-php_cli="/usr/local/bin/php"
+php_cli=$(which php)
```
Comment (guarinogabriel) [inline mac]: @andypham Awesome!

**Comment by guarinogabriel** [comment]
@andypham I added some comments on your PR. Could you please take a look?

Thanks for your contribution!

File: mac (hunk 2 above)
Comment (waqasrana) [inline mac]: both commands work fine for me (same for email). Just wondering whats causing .* one to break for you?

Comment (guarinogabriel) [inline mac]: @waqasrana That is correct the asterisk is breaking the command for me:
`zsh: no matches found: user.name=.*`

**Comment by guarinogabriel** [comment]
@andypham I am closing this PR since your changes have been manually included on the CLI.
Thanks so much for your contribution!

File: mac (hunk 3 above)
Comment (andypham) [inline mac]: Is probably the EDITOR env var is not set. This should guard that condition ${EDITOR:-nano}

**Comment by andypham** [comment]
@guarinogabriel Sorry for the slow respond as I was away for the 4th July holiday. I added the comment for the EDITOR env variable.

**Pull request closed without merging**